microsoftgraph/microsoft-graph-explorer-v4
# Fix: ctrl-v error in url

## Overview
Pasting something on the url was causing the controls to get lost

Fixes #428 

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Changes the share query function to use the standardized 'parseUrl' method

## Testing Instructions

* Copy an invalid item
* Clear query input
* Paste using Ctrl+V and notice that the page does not error out
